client/src/components/History/CurrentHistory/HistoryNavigation.vue

```diff
@@ -264,16 +274,14 @@ function userTitle(title: string) {
             id="delete-history-modal"
             title="Delete History?"
             title-tag="h2"
-            @ok="historyStore.deleteHistory(history.id, false)">
-            <p v-localize>Really delete the current history?</p>
-        </BModal>
-
-        <BModal
-            id="purge-history-modal"
-            title="Permanently Delete History?"
-            title-tag="h2"
-            @ok="historyStore.deleteHistory(history.id, true)">
-            <p v-localize>Really delete the current history permanently? This cannot be undone.</p>
+            @ok="onDelete"
+            @show="purgeHistory = false">
+            <p v-localize>
+                Do you also want to purge the history <i class="ml-1">{{ history.name }}</i>
```

Comment:
```suggestion
                Do you also want to permanently delete the history <i class="ml-1">{{ history.name }}</i>
```

I think we decided to stick to the less technical 'permanently deleted' vs 'purged'.

Reply:
Yes, this is more understandable for the user, thanks!